griptape/drivers/text_to_speech/elevenlabs_text_to_speech_driver.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Optional, Any
+
+from attr import define, field, Factory
+
+from griptape.artifacts.audio_artifact import AudioArtifact
+from griptape.drivers import BaseTextToSpeechDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from elevenlabs.client import ElevenLabs
+
+
+@define
+class ElevenLabsTextToSpeechDriver(BaseTextToSpeechDriver):
+    api_key: str = field(kw_only=True, metadata={"serializable": True})
+    client: Any = field(
+        default=Factory(
+            lambda self: import_optional_dependency("elevenlabs.client").ElevenLabs(api_key=self.api_key),
+            takes_self=True,
+        ),
+        kw_only=True,
+        metadata={"serializable": True},
+    )
+    voice: str = field(kw_only=True, metadata={"serializable": True})
+    output_format: str = field(default="mp3_44100_128", kw_only=True, metadata={"serializable": True})
```

Comment:
nit: maybe move this default to a top level constant?

Reply:
not sure what the guideline is for inline defaults vs top-level constants. seems like its done both ways.